client/src/app/create-deployment/create-deployment.component.ts

```diff
@@ -19,7 +19,7 @@ export class CreateDeploymentComponent implements OnInit {
   deploymentForm!: FormGroup;
   serverSizes: string[] = [];
   amis: string[] = [];
-  regions: Region[] = [Region.AP_SOUTHEAST_3];
+  regions: string[] = [];
```

Comment:
Do we still want to set the type to list?

Reply:
I think there was a reason for that last time since it will give out an error if not, I'll need to recheck this.

Reply:
Alright, the type is fixed to string instead of list